content/_data/changelogs/weekly.yml

```diff
@@ -8863,6 +8863,60 @@
   # pull: 4927 (PR title: Added localizaton for devsecops=DevSecOps)
   # pull: 4930 (PR title: [JENKINS-63639] Update commons-io to 2.8.0)
 
+- version: '2.258'
+  date: 2020-09-22
+  changes:
+  - type: major bug
+    category: regression
+    pull: 4942
+    issue: 62661
+    authors:
+    - res0nance
+    message: |-
+      Migrate the view status filter from views in previous Jenkins releases (regression in 2.240).
+  - type: rfe
+    category: rfe
+    pull: 4926
+    issue: 63592
+    authors:
+    - fqueiruga
+    message: |-
+      Revamp the empty state screens.
+      The revised strings are currently only available in English.
```

Comment:
is this relevant?

Reply:
I was undecided on this one.  A user running the localized version of Jenkins will now see English strings in the page shown when no jobs are configured, even if they previously saw localized strings.  I assumed @fqueiruga wanted to use this message to inform those users that this is a known condition.

As a counterpoint, it is describing the same effect that users of localized versions see each time a string is modified.  Thus, it may be telling them something that is already common experience and fully expected.  I'm willing to remove the string.

I was more concerned about the clarity of the phrase "empty state screens", but didn't have a good suggestion for an alternative phrasing.

Reply:
```suggestion
```

not worth a changelog entry imo, a user is very unlikely to ever see this, it's the admin who sets it up the first time.